src/test/scala/cromwell/binding/values/WdlValueSpec.scala

```diff
@@ -0,0 +1,101 @@
+package cromwell.binding.values
+
+import java.nio.file.Paths
+
+import cromwell.binding.{WdlExpression, WdlNamespace}
+import cromwell.util.SampleWdl
+import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.{FlatSpec, Matchers}
+
+class WdlValueSpec extends FlatSpec with Matchers {
+
+  import TableDrivenPropertyChecks._
+
+  behavior of "WdlValue"
+
+  val wdlValueRawStrings = Table(
+    ("wdlValue", "rawString"),
+    (WdlBoolean(false), "false"),
+    (WdlBoolean(true), "true"),
```

Comment:
These two lines beget IntelliJ warnings for unnamed boolean parameters, fortunately `WdlBoolean.True` and `WdlBoolean.False` can help with that.  :smile: